features/navigator-namespace.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Navigator Namespace</title>
+</head>
+<body>
+<p><a href="../index.html">[Home]</a></p>
+
+<p>Using the new DOM API, <code>navigator.ddg</code>, to provide signals about the presence of DDG products</p>
+
+<p id="ddg"></p>
+<p id="ddg-extension"></p>
+
+<script>
+    const hasNamespace = 'ddg' in navigator;
+    let isExtension = false;
+
+    if (hasNamespace) {
+        if (typeof navigator.ddg.isExtension === 'function') {
+            isExtension = navigator.ddg.isExtension();
+        }
+    }
+
+    window.results = {
+        page: 'navigator-namespace',
+        date: null,
```

Comment:
```suggestion
        date: (new Date()).toUTCString(),
```

let's have a date in case someone stores the results